package.json

```diff
@@ -8,136 +8,71 @@
     "url": "https://github.com/kubevirt/web-ui-components.git"
   },
   "main": "dist/index.js",
-  "devDependencies": {
-    "@storybook/addon-actions": "^3.4.10",
-    "@storybook/addon-options": "^3.4.10",
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "@storybook/storybook-deployer": "^2.1.0",
-    "babel-core": "6.26.0",
-    "babel-eslint": "7.2.3",
-    "babel-jest": "20.0.3",
-    "babel-loader": "7.1.2",
-    "babel-preset-react-app": "^3.1.1",
-    "babel-runtime": "6.26.0",
-    "chalk": "1.1.3",
-    "coveralls": "^3.0.2",
-    "css-loader": "0.28.7",
-    "dotenv": "4.0.0",
-    "dotenv-expand": "4.2.0",
-    "enzyme": "^3.5.1",
-    "enzyme-adapter-react-16": "^1.3.1",
-    "enzyme-to-json": "^3.3.4",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-config-standard-jsx": "^4.0.2",
-    "eslint-config-standard-react": "^5.0.0",
-    "eslint-import-resolver-webpack": "^0.9.0",
-    "eslint-loader": "^2.1.0",
-    "eslint-plugin-import": "^2.10.0",
-    "eslint-plugin-jest": "^21.15.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-patternfly-react": "*",
-    "eslint-plugin-prettier": "^2.4.0",
-    "eslint-plugin-promise": "^3.7.0",
-    "eslint-plugin-react": "^7.7.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "1.1.5",
-    "fs-extra": "3.0.1",
-    "html-webpack-plugin": "2.29.0",
-    "jest": "20.0.4",
-    "lodash": "^4.17.11",
-    "object-assign": "4.1.1",
-    "patternfly-react": "^2.14.0",
-    "postcss-flexbugs-fixes": "3.2.0",
-    "postcss-loader": "2.0.8",
-    "prettier": "^1.14.2",
-    "prettier-eslint": "^8.8.1",
-    "promise": "8.0.1",
-    "prop-types": "^15.6.2",
-    "raf": "3.4.0",
-    "react": "^16.4.2",
-    "react-dev-utils": "^5.0.1",
-    "react-dom": "^16.4.2",
-    "resolve": "1.6.0",
-    "style-loader": "0.19.0",
-    "stylelint": "^8.4.0",
-    "stylelint-config-standard": "^18.2.0",
-    "sw-precache-webpack-plugin": "0.11.4",
-    "url-loader": "0.6.2",
-    "webpack": "3.8.1",
-    "webpack-dev-server": "2.9.4",
-    "webpack-manifest-plugin": "1.3.2",
-    "whatwg-fetch": "2.0.3"
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "lint": "yarn lint-js && yarn lint-css",
+    "lint-js": "yarn lint-js-browser && yarn lint-js-node",
```

Comment:
I find `lint:js` more common than `lint-js` (colon vs. dash)

Reply:
Does this mean you also prefer e.g. `lint:js:browser` over `lint-js-browser`?

Reply:
If we really need such deep hierarchy and so many targets, than yes. *But this is nitpicking.* Just pointing to what is more common elsewhere.

My other preference is to keep the number of targets (scripts) at reasonably low number - more or less just those which are really called manually or as a "code sharing" if content of targets overlaps.
I find it nice if I can understand what to call from `scripts` at the first glance.

Reply:
> If we really need such deep hierarchy and so many targets, than yes.

OK, I'll go with `lint:js:browser` format. [patternfly-react](https://github.com/patternfly/patternfly-react/blob/master/package.json) project uses it too.

> My other preference is to keep the number of targets (scripts) at reasonably low number - more or less just those which are really called manually or as a "code sharing" if content of targets overlaps.

Agreed.